frontend/public/components/chargeback.tsx

```diff
@@ -317,23 +233,28 @@ class ReportData extends React.Component<ReportDataProps, ReportDataState> {
 
   render() {
     const {obj} = this.props;
-    const phase = _.get(obj, ['status', 'phase']);
-
-    const {data, reduceBy, sortBy, orderBy, keys, rows, maxValues, totals, inFlight, error} = this.state;
+    const conditions = _.get(obj, 'status.conditions');
+    const isFinished = _.some(conditions, {reason: 'Finished'});
```

Comment:
This doesn't look right. Can we use...?

```suggestion
    const isFinished = _.some(conditions, {type: 'Running', status: 'False'});
```

Reply:
If we really need to check a specific `reason`, we should also check `type` and `status` in addition to `reason`. I'd prefer to avoid that if we can, though. Better not to check specific `reason` values.

Reply:
Ok, that makes sense, fixed locally, not yet pushed

Reply:
Fixed, pushed